src/Features/Core/Portable/CodeRefactoringHelpers.cs

```diff
@@ -131,5 +171,40 @@ internal static class CodeRefactoringHelpers
 
             return TextSpan.FromBounds(start, end);
         }
+
+        /// <summary>
+        /// Strips leading and trailing whitespace from <paramref name="span"/>.
+        /// </summary>
+        /// <remarks>
+        /// Returns unchanged <paramref name="span"/> in case <see cref="TextSpan.IsEmpty"/>.
+        /// Returns Span of Lenght 1 in case it contains only whitespace.
```

Comment:
Any specific reason why we don’t return an empty span for this case? Not sure it matters but just was curious.

Reply:
It's potentially one extra check but you're right, thinking about it it's the sensible thing to do. Will fix tomorrow.